sdk/appconfiguration/Azure.ResourceManager.AppConfiguration/src/Azure.ResourceManager.AppConfiguration.csproj

```diff
@@ -8,5 +8,6 @@
     </Description>
     <PackageTags>azure;management;appconfiguration</PackageTags>
     <NoWarn>$(NoWarn);AZC0001;AZC0008;</NoWarn>
+    <IsOldTrack2>true</IsOldTrack2>
```

Comment:
Looks like this property is only used to exclude the mgmtcore shared sources should we instead name it as such? Perhaps 'ExcludeMgmtCoreShared` or something like that?

Reply:
renamed